bin/generate-pot-with-wpcli.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script generates a POT file for a WordPress plugin or theme using WP-CLI.
+# We are using wp-scripts to generate plugin zip which bundle files defined in 'package.json' in files param.
+# https://developer.wordpress.org/block-editor/packages/packages-scripts/#files
+# This script will run with npm run makepot command.
+#
+# @since x.x.x
+
+if ! command -v wp &>/dev/null; then
+	echo "Error: wp cli could not be found. Please install wp cli and try again."
+	exit
+fi
+
+# Get the list of files and directories which bundle in final zip file.
+FILES=$(node -p "require('./package.json').files.join(',')")
```

Comment:
This includes `"lang/**/*",` can this cause problems?

Reply:
@peterwilsoncc it will not cause an issue because WP CLI only sniffs javascript and  PHP files.

> Scans PHP, Blade-PHP and JavaScript files for translatable strings, as well as theme stylesheets and plugin files if the source directory is detected as either a plugin or theme.
ref: https://developer.wordpress.org/cli/commands/i18n/make-pot/


But I excluded `lang\` directory: https://github.com/10up/distributor/pull/1014/commits/40ee454bce484fbe49f2c54ecf87610ac375d83a